bin/cli.js

```diff
@@ -3,6 +3,7 @@
 const Liftoff = require('liftoff');
 const interpret = require('interpret');
 const path = require('path');
+const pkgDir = require('pkg-dir');
```

Comment:
pkg dir is being used to find the package.json of the project in which the clie is executed.